modules/test/playwright/tests/frontend-taglib/searchPaginator.spec.ts

```diff
@@ -5,40 +5,36 @@
 
 import {Locator, expect, mergeTests} from '@playwright/test';
 
-import {apiHelpersTest} from '../../../fixtures/apiHelpersTest';
-import {featureFlagsTest} from '../../../fixtures/featureFlagsTest';
-import {isolatedSiteTest} from '../../../fixtures/isolatedSiteTest';
-import {loginTest} from '../../../fixtures/loginTest';
-import getRandomString from '../../../utils/getRandomString';
-import getPageDefinition from '../../layout-content-page-editor-web/utils/getPageDefinition';
-import getWidgetDefinition from '../../layout-content-page-editor-web/utils/getWidgetDefinition';
+import {featureFlagsTest} from '../../fixtures/featureFlagsTest';
+import {isolatedSiteTest} from '../../fixtures/isolatedSiteTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {taglibSamplePageTest} from './fixtures/taglibSamplePageTest';
 
 export const test = mergeTests(
-	apiHelpersTest,
 	featureFlagsTest({
 		'LPS-178052': true,
 	}),
 	isolatedSiteTest,
-	loginTest()
+	loginTest(),
+	taglibSamplePageTest
 );
 
+const linkName = 'Search Paginator';
+
 test(
 	'Search Paginator dropdown generates page links on scrolling',
 	{tag: '@LPD-37458'},
-	async ({apiHelpers, page, site}) => {
+	async ({page, site, taglibSamplePage}) => {
 		let dropdownMenuHandler: Locator;
 
-		await test.step('Create a content site and the frontend taglib sample widget', async () => {
-			const widgetDefinition = getWidgetDefinition({
-				id: getRandomString(),
-				widgetName: 'com_liferay_sample_web_portlet_SamplePortlet',
+		await test.step('Create a content site and the taglib sample widget', async () => {
```

Comment:
I think `frontend taglib sample widget` is correct because that is the end-user naming we already had in master. I would leave as it was.